[NavigatorIOS] Allow translucent on NavigatorIOS

ojkelly opened the pull request:
Hi,

I've updated the NavigatorIOS component to allow setting the translucent property.

usage is:
```
            <NavigatorIOS
                translucent={false}
            />
```


This is my first contrib to react-native, so apologies if I've missed something.

Cheers,
Owen

brentvatne commented:
@ojkelly - this looks good to me, can you add an example to UIExplorer? :smile:

a2 commented on React/Views/RCTNavItem.h:
```diff
@@ -22,6 +22,8 @@
 @property (nonatomic, strong) UIColor *barTintColor;
 @property (nonatomic, strong) UIColor *titleTextColor;
+@property (nonatomic, assign) BOOL translucent;
+
```
Nit: remove newline

a2 commented:
Looks good as well. If you can remove that one newline (sorry, OCD :wink:) and add an example as @brentvatne suggests, I'll merge this. :)

ojkelly commented:
Hey @a2 sorry for the delay, I was away.

I've removed that newline, and added an example. After looking at both NavigatorIOS examples, I thought NavigatorIOSColorsExample.js made the most sense to put it. I can move it if we need though.
